doc/build-windows-mingw.md

```diff
@@ -0,0 +1,250 @@
+
+Download and unpack http://fukuchi.org/works/qrencode/qrencode-3.4.4.tar.gz inside your deps folder 
+then configure and make:
+
+
+cd /c/deps/qrencode-3.4.4
+
+LIBS="../libpng-1.6.16/.libs/libpng.a ../../mingw32/i686-w64-mingw32/lib/libz.a" \
+png_CFLAGS="-I../libpng-1.6.16" \
+png_LIBS="-L../libpng-1.6.16/.libs" \
+configure --enable-static --disable-shared --without-tools
+
+make
+
+
+
+2.7 Qt 5 libraries:
+Qt must be configured with ssl and zlib support.
+Download and unpack Qt base and tools sources:
+http://download.qt-project.org/official_releases/qt/5.3/5.3.2/submodules/qtbase-opensource-src-5.3.2.7z
+http://download.qt-project.org/official_releases/qt/5.3/5.3.2/submodules/qttools-opensource-src-5.3.2.7z
+Then from a windows command prompt (note that the following assumes qtbase has been unpacked to C:\Qt\5.3.2 and 
+qttools have been unpacked to C:\Qt\qttools-opensource-src-5.3.2):
+
+
+set INCLUDE=C:\deps\libpng-1.6.16;C:\deps\openssl-1.0.1k\include
+set LIB=C:\deps\libpng-1.6.16\.libs;C:\deps\openssl-1.0.1k
+
+cd C:\Qt\5.3.2
+configure.bat -release -opensource -confirm-license -static -make libs -no-sql-sqlite -no-opengl -system-zlib -qt-pcre -no-icu -no-gif -system-libpng -no-libjpeg -no-freetype -no-angle -no-vcproj -openssl -no-dbus -no-audio-backend -no-wmf-backend -no-qml-debug
+mingw32-make -j4
+
+set PATH=%PATH%;C:\Qt\5.3.2\bin
+set PATH=%PATH%;C:\Qt\qttools-opensource-src-5.3.2
+
+cd C:\Qt\qttools-opensource-src-5.3.2
+qmake qttools.pro
+mingw32-make -j4
+
+Note: consider using -j switch with mingw32-make to speed up compilation process. On a quad core -j4 or -j5 should give the best results.
+
+
+then 
+
+make a BOOST_ROOT environment variable.  best to place it in your mysys.bat file which should
+be on your desktop now.  that way it's there whenever you startup the shell.
+Enter the following in the mysys shell or add it to the top (after the comment section) of the mysys.bat file.
+
+set BOOST_ROOT=/c/deps/boost_1_61_0
+
+
+
+Build bitcoin
+-------------
+
+From the mysys shell prompt cd to the root of the source code directory
+and type the following command (for example, cd c:/bitcoin and NOT c:/bitcoin/src):
+
+./autogen.sh
+
+Once autogen completes, from the mysys shell prompt enter the following:
+
+
+CPPFLAGS="-I/c/deps/db-4.8.30.NC/build_unix \
+-I/c/deps/openssl-1.0.1k/include \
+-I/c/deps/libevent-2.0.22/include \
+-I/c/deps \
+-I/c/deps/protobuf-2.6.1/src \
+-I/c/deps/libpng-1.6.16 \
+-I/c/deps/qrencode-3.4.4" \
+LDFLAGS="-L/c/deps/db-4.8.30.NC/build_unix \
+-L/c/deps/openssl-1.0.1k \
+-L/c/deps/libevent-2.0.22/.libs \
+-L/c/deps/miniupnpc \
+-L/c/deps/protobuf-2.6.1/src/.libs \
+-L/c/deps/libpng-1.6.16/.libs \
+-L/c/deps/qrencode-3.4.4/.libs" \
+BOOST_ROOT=/c/deps/boost_1_61_0 \
+./configure \
+--disable-upnp-default \
+--disable-tests \
+--with-qt-incdir=/c/Deps/Qt/5.3.2/include \
+--with-qt-libdir=/c/Deps/Qt/5.3.2/lib \
+--with-qt-plugindir=/c/Deps/Qt/5.3.2/plugins \
+--with-qt-bindir=/c/Deps/Qt/5.3.2/bin \
```

Comment:
There is an inconsistency with the location of the Qt root folder here versus in the section where Qt is built.  Here it is below `C:\Deps\` with the rest of the dependencies, however in the build section for Qt it is directly under `C:\Qt\`.

For my build I changed this location, however for consistency it's probably better to use the `C:\deps\` location as that's where the rest of the dependencies are located.